extras/docker/Dockerfile

```diff
@@ -41,6 +41,7 @@ COPY --from=build /build/quiche/target/release/examples/client .
 COPY --from=build /build/quiche/target/release/examples/server .
 COPY --from=build /build/quiche/target/release/examples/http3-client .
 COPY --from=build /build/quiche/target/release/examples/http3-server .
+ENV RUST_LOG=trace
```

Comment:
It may impact perf test (if added). If no plan to do in the near future it will be good.

Reply:
I spoke with the QNS people, and they say that implementations shouldn't expect to be CPU-limited by the interop or QNS tests (that is, logging everything should be fine, e.g. quicly does that as well).

We can reconsider this later on if it becomes a problem though.